Jenkinsfile-new

```diff
@@ -152,6 +152,7 @@ node ('master') {
   packagePush = false
   specialBranch = false
   parallelism = 0
+  useBTF = false
 
   if (scmVars.GIT_LOCAL_BRANCH in ["master","develop","dev"] || scmVars.CHANGE_BRANCH_LOCAL in ["develop","dev"])
```

Comment:
Shall we get rid of 'dev' branch mentions since we moved back to good old `develop` ?